rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -198,13 +204,24 @@ def get_airdrop_data(name: str, data_dir: Path) -> Tuple[Iterator, TextIO]:
     if not filename.is_file():
         # if not cached, get it from the gist
         try:
-            request = requests.get(url=AIRDROPS[name][0], timeout=DEFAULT_TIMEOUT_TUPLE)
+            response = requests.get(url=AIRDROPS[name][0], timeout=DEFAULT_TIMEOUT_TUPLE)
         except requests.exceptions.RequestException as e:
             raise RemoteError(f'Airdrops Gist request failed due to {str(e)}') from e
-
-        with open(filename, 'w') as f:
-            f.write(request.content.decode('utf-8'))
-
+        try:
+            content = response.text
+            if (
+                not csv.Sniffer().has_header(content) or
+                len(response.content) < SMALLEST_AIRDROP_SIZE
+            ):
+                raise csv.Error
+            with open(filename, 'w', newline='') as f:
```

Comment:
hmm didn't we say to not do that? This will mean that universal newlines won't be translated to the OS level at writing.

So what will happen is it will be stored in the disk as whatever the bytes of the original file were. Which is probably okay for the files we got, but it's really not needed. Just leave the default of `None`. We want the translation.

>When writing output to the stream, if newline is None, any '\n' characters written are translated to the system default line separator, os.linesep. If newline is '' or '\n', no translation takes place